src/main/java/duke/UI.java

```diff
@@ -0,0 +1,92 @@
+package duke;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * An input/output handler that deals with interaction with the user.
+ */
+public class UI {
```

Comment:
You might want to change this class name is "Ui" to follow coding standard for acronym names